src/Domain.jl

```diff
@@ -1599,6 +1599,49 @@ function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=
     return jac
 end
 
+function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=nothing) where {Q3<:AbstractArray,Q2,Q<:AbstractArray,U<:AbstractArray,W,Z<:Real,D<:ConstantVDomain}
+    ns,cs,T,P,V,C,N,mu,kfs,krevs,Hs,Us,Gs,diffs,Cvave = calcthermo(domain,y,t,p)
+    jacobiany_ns!(jac,y,p,t,domain,[],cs,kfs,krevs,nothing)
+    
+    cpdivR,hdivRT1,sdivR = calcHSCpdless(domain.phase.vecthermo,T)
```

Comment:
It should be easy for us to add cpdivR from the calcthermo functions with virtually no computational cost (since the vector is already created in calcthermo for these domains).

Reply:
I modified all the calcthermo functions to output cpdivR when T is not constant or parametrized. For isothermal or parametrized T Domain, it outputs Array{Float64,1}() instead.